### DISCUSSION TITLE
How can participatory mapping and critical cartography be improved?

### DISCUSSION THREAD

**Discussion by lvalles10** [albertkun/21S-ASIAAM-191A#57] [Past Thinking Caps]
One particularly interesting addition Mariah brought to our group's conversation on participatory mapping and critical cartography was the importance of data sovereignty, which is an emerging movement on the part of native folks to collect, own, and provide their own data for their maps. For example, Mariah brought up how Maori folks have started classifying data sets with their own language so as to protect their data sovereignty and the maps that are created from it. 

One drawback to data sovereignty (as well as critical cartography and participatory mapping at large) we did discuss as a group was the "catch-22" of not being able to integrate data sovereignty into government managed data systems such as the Census. On one hand data sovereignty allows native people to propagate their own data and protect their authentic narratives; on the other, data sovereignty outside of the "official" bureaucracy of our government isn't counted in important measures that come from the Census, which reduces funding and representation for native folks if they aren't correctly counted. 

At large, the practices of participatory mapping and critical cartography can be improved by inviting more people into the fold; by diversifying the perspectives of map-makers and further integrating "map artists," the rigid hierarchy and oppressive structures that continue to dominate mapping can be dismantled. By integrating practices such as data sovereignty, participatory mapping and critical cartography can better reflect the needs and interests of civil society (especially its most marginalized). This is easier said than done, but awareness and discussion on these issues should serve to improve their practice. Excited to hear other folks' thoughts on this!

**Comment by rachan2023** [comment]
You bring up some great points! In order to improve participatory mapping and critical cartography, people should learn about the consequences of map-making on marginalized communities and focus on providing people open-source mapping tools to create their own maps in order to  write their own story. Institutions and organizations, such as the government and the Census, can collect data and incorporate their own biases to make maps that fit their political and social agendas, which leads to the erasure of communities. As you've mentioned, the Maori people are trying to protect their data by classifying their data with their own language, which is important because they are taking back their power to write their own story. When GIS and map-making tools are accessible to the public, we are able to see different perspectives and information that may be left out from one map. With multitudes of different maps, people are able to think critically and compare the "standard" world political map with the maps that people have created to see how the world political map is oppressive, especially for marginalized communities.
